appscale/tools/local_state.py

```diff
@@ -420,6 +420,29 @@ def update_local_metadata(cls, options, db_master, head_node):
 
 
   @classmethod
+  def clean_local_metadata(cls, keyname):
+    try:
+      with open(cls.get_locations_json_location(keyname), 'r') as file_handle:
+        file_contents = yaml.safe_load(file_handle.read())
```

Comment:
I'm curious about this yaml load. Do we ever expect this file to contain non-json data?